webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/uploading/UploadService.scala

```diff
@@ -133,6 +135,10 @@ class UploadService @Inject()(dataSourceRepository: DataSourceRepository,
         redisKeyForDataSourceId(reserveUploadInformation.uploadId),
         Json.stringify(Json.toJson(DataSourceId(reserveUploadInformation.name, reserveUploadInformation.organization)))
       )
+      _ <- runningUploadMetadataStore.insert(
+        Json.stringify(Json.toJson(DataSourceId(reserveUploadInformation.name, reserveUploadInformation.organization))),
+        reserveUploadInformation.uploadId
+      )
       _ <- runningUploadMetadataStore.insert(
         redisKeyForLinkedLayerIdentifier(reserveUploadInformation.uploadId),
```

Comment:
Now we are inserting two different types of data into the same set. Maybe this could be unified? Otherwise, it may be better practice to have two stores and do the set existence test on each specific one?